modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -940,6 +940,57 @@ public void run() {
         StatusThreadPoolExecutor.getInstance().execute(r);
     }
 
+    @ReactMethod
+    public void getConnectionStringForBootstrappingAnotherDevice(final String configJSON, final Callback callback) throws JSONException {
+        Log.d(TAG, "getConnectionStringForBootstrappingAnotherDevice");
+         final JSONObject jsonConfig = new JSONObject(configJSON);
+         final String keyUID = jsonConfig.getString("keyUID");
+         final String keyStorePath = this.getKeyStorePath(keyUID);
+//          final String keyStorePath = pathCombine(this.getNoBackupDirectory(), "/keystore");
+         jsonConfig.put("keystorePath", keyStorePath);
+
+        if (!checkAvailability()) {
+            callback.invoke(false);
+            return;
+        }
+
+        Runnable r = new Runnable() {
```

Comment:
not particularly a fan of variables where the name doesn't explain its function. Consider renaming this to something more explanative :)

Reply:
I understand, I have just followed the pattern thats used in most of the functions in this file, I will rename this :)

Reply:
for sure, my comments are just suggestions. Do whatever you think is best 👍

Reply:
I went ahead and name it `runnableTask`